src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/update/SetOperation.java

```diff
@@ -38,8 +38,22 @@ public static SetOperation constructSet(ObjectNode args) {
    * $setOnInsert)
    */
   public static SetOperation constructSet(String filterPath, JsonNode value) {
+    return constructSet(filterPath, value, false);
+  }
+
+  /**
+   * Override method used to create an operation that $sets a single property (never used for
+   * $setOnInsert)
+   *
+   * @param filterPath the path to set
+   * @param value the value to set
+   * @param skipValidation whether to skip validation, used for internl purposes for so _id and
```

Comment:
Having skipValidation here is weird IMO
If this is for internal usage, we could create a new construct method with a better name.

Reply:
Yuqi Resolve: can be resolved, added a TODO